### PR TITLE
fix(FEC-12562) - ignore ks configuration on kava plugin incase of ottAnalytics exists on player

### PR DESCRIPTION
**Pull request by giladna**

### CODE REVIEW

File: src/kaltura-player.js [hunk 1]
```diff
@@ -898,6 +898,9 @@ class KalturaPlayer extends FakeEventTarget {
     const middlewares = [];
     const uiComponents = [];
     const plugins = [];
+    if ('kava' in pluginsConfig && 'ottAnalytics' in pluginsConfig && 'ks' in pluginsConfig.kava) {
+      delete pluginsConfig.kava.ks;
```
Comment (OrenMe) [inline src/kaltura-player.js:902]: Move to helper function

Comment (giladna) [reply src/kaltura-player.js]: done

**Review by OrenMe — DISMISSED** [review]

File: src/kaltura-player.js [hunk 2]
```diff
@@ -898,6 +898,9 @@ class KalturaPlayer extends FakeEventTarget {
     const middlewares = [];
     const uiComponents = [];
     const plugins = [];
```
Comment (SivanA-Kaltura) [inline src/kaltura-player.js:900]: I would split into 2 default config files (ovp, ott) as we've talked.

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/kaltura-player.js (hunk 2 above)
Comment (giladna) [reply src/kaltura-player.js]: done thanks

**Comment by giladna** [comment]
opening new one

**Pull request closed without merging**